codespell_lib/data/dictionary.txt

```diff
@@ -10277,6 +10286,10 @@ decendentant->descendant
 decendentants->descendants
 decendents->descendents, descendants,
 decending->descending
+decern->discern
+decerned->discerned
+decerning->discerning
+decerns->discerns
```

Comment:
I would expect French speakers to use "decern" instead of "award". Just adding this for future reference: no need to change anything, because that's not a spelling issue as such.